purchases/src/main/kotlin/com/revenuecat/purchases/VirtualCurrencyInfo.kt

```diff
@@ -0,0 +1,34 @@
+package com.revenuecat.purchases
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * A class representing information about a virtual currency in the app.
+ *
+ * @property balance The current balance of the virtual currency.
+ */
+@Parcelize
+data class VirtualCurrencyInfo internal constructor(
```

Comment:
Do we want to mark this with `@ExperimentalPreviewRevenueCatPurchasesAPI` for now? (Just checking)

Reply:
I think this makes sense! I've added `@ExperimentalPreviewRevenueCatPurchasesAPI` in 1be37ca848c21a586e1d63a960e791be39bdda68.

This required me to opt-in to it in a few places internally, are there any downsides to that? You can see all of them in the commit (1be37ca848c21a586e1d63a960e791be39bdda68). If we don't like it, I can revert the commit.